addons/account_edi_facturx/i18n/account_edi_facturx.pot

```diff
@@ -20,6 +20,12 @@ msgstr ""
 msgid "Display Name"
 msgstr ""
 
+#. module: account_edi_facturx
+#: code:addons/account_edi_facturx/models/account_edi_format.py:0
+#, python-format
+msgid "Display the currency"
+msgstr ""
+
```

Comment:
was previously missing?

Reply:
It is the text I added on the button of the redirect warning